<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Submit Checklist</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Location</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QRadioButton" name="rb_existingLoc">
                 <property name="text">
                  <string>Existing</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="existingLocEntry"/>
               </item>
               <item row="1" column="0">
                <widget class="QRadioButton" name="rb_newLoc">
                 <property name="text">
                  <string>New</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="newLocEntry"/>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>State:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="submitStates">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>County:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="submitCounties"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Date / Time</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCalendarWidget" name="calendarWidget"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Start:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="startTime"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>End:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="endTime"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Protocol</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QRadioButton" name="rb_incidental">
               <property name="text">
                <string>Incidental</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="rb_stationary">
               <property name="text">
                <string>Stationary</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="rb_traveling">
               <property name="text">
                <string>Traveling</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Observers:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Distance (miles):</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="numObservers">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleSpinBox" name="distanceTraveled">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Species</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="1">
              <widget class="QComboBox" name="existingSpeciesEntry">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="speciesCount">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pb_addSpecies">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="newSpeciesEntry"/>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="rb_existingSpecies">
               <property name="text">
                <string>Existing</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="rb_newSpecies">
               <property name="text">
                <string>New</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="checklistView"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pb_submitChecklist">
               <property name="text">
                <string>Submit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_cancelChecklist">
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Explore Database</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Database Filtering</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QRadioButton" name="rb_firstSightings">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>First Records</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Species:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QPushButton" name="pb_applyFilter">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Apply Filter</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="rb_allSightings">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>All Records</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>States:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Locations:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QComboBox" name="exploreLocations"/>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="exploreSpecies"/>
           </item>
           <item row="1" column="5">
            <widget class="QPushButton" name="pb_resetFilter">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Reset Filter</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="rb_lastSightings">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Last Records</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="exploreStates"/>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>End Date:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Start Date:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QDateEdit" name="exploreEndDate">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QDateEdit" name="exploreStartDate">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="exploreResults"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>rb_existingLoc</tabstop>
  <tabstop>rb_newLoc</tabstop>
  <tabstop>existingLocEntry</tabstop>
  <tabstop>newLocEntry</tabstop>
  <tabstop>submitStates</tabstop>
  <tabstop>submitCounties</tabstop>
  <tabstop>calendarWidget</tabstop>
  <tabstop>startTime</tabstop>
  <tabstop>endTime</tabstop>
  <tabstop>rb_incidental</tabstop>
  <tabstop>rb_stationary</tabstop>
  <tabstop>rb_traveling</tabstop>
  <tabstop>numObservers</tabstop>
  <tabstop>distanceTraveled</tabstop>
  <tabstop>rb_existingSpecies</tabstop>
  <tabstop>existingSpeciesEntry</tabstop>
  <tabstop>speciesCount</tabstop>
  <tabstop>pb_addSpecies</tabstop>
  <tabstop>pb_submitChecklist</tabstop>
  <tabstop>pb_cancelChecklist</tabstop>
  <tabstop>rb_newSpecies</tabstop>
  <tabstop>newSpeciesEntry</tabstop>
  <tabstop>checklistView</tabstop>
  <tabstop>rb_allSightings</tabstop>
  <tabstop>rb_firstSightings</tabstop>
  <tabstop>rb_lastSightings</tabstop>
  <tabstop>exploreSpecies</tabstop>
  <tabstop>exploreStates</tabstop>
  <tabstop>exploreLocations</tabstop>
  <tabstop>exploreStartDate</tabstop>
  <tabstop>exploreEndDate</tabstop>
  <tabstop>pb_resetFilter</tabstop>
  <tabstop>pb_applyFilter</tabstop>
  <tabstop>exploreResults</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
